config/babel.config.js

```diff
@@ -59,6 +58,7 @@ module.exports = function(api) {
             ],
           },
           useBuiltIns: 'entry',
+          corejs: '3.9',
```

Comment:
We were getting a warning without this, so I figured I'd throw it in there.
![image](https://user-images.githubusercontent.com/12970166/108574917-7020ec00-72cd-11eb-8a5e-a6e59f6043d3.png)